doc/index.rst

```diff
@@ -152,7 +152,7 @@ Optional environment variables
 Note that some of these currently tied only to a single option, but we still leave them for the future when more options become available (e.g. ament with BUILDER).
 
 * `ABICHECK_MERGE` (default: not set): Used only when `ABICHECK_URL` is set. For travis it can be set to 'auto' to auto-detect pull requests. If set to 'true' the merge parent (see `Terminology section <#terminology>`_) will be checked against.
-* `ABICHECK_URL` (default: not set): Run binary compatibility check with `ABICC <https://github.com/lvc/abi-compliance-checker>`_. The URL should point to a baseline archive (\*.tar.\*,\*.zip, \*.tgz or \*.tbz2), see more in `the ABI checks section <#abi-checks>`_)
+* `ABICHECK_URL` (default: not set): Run binary compatibility check with `ABICC <https://github.com/lvc/abi-compliance-checker>`_. The URL should point to a baseline of either a branch or an archive (\*.tar.\*,\*.zip, \*.tgz or \*.tbz2). See more in `the ABI checks section <#abi-checks>`_)
```

Comment:
No, it still must be an archive.

Reply:
But what about `github:ros-industrial-release/ros_canopen-release#upstream`?

Reply:
This is a shortcut form that [gets expanded](https://github.com/ipa-mdl/industrial_ci/blob/feature/abi-check/industrial_ci/src/tests/abi_check.sh#L95)  to  https://github.com/ros-industrial-release/ros_canopen-release/archive/upstream.tar.gz